lib/trackler/track.rb

```diff
@@ -81,8 +81,11 @@ def test_pattern
       end
     end
 
-    def docs(image_path = DEFAULT_IMAGE_PATH)
-      OpenStruct.new(docs_by_topic(image_path))
+    def docs(positional_image_path_which_is_deprecated = nil, image_path: nil)
+      if positional_image_path_which_is_deprecated
+        warn "DEPRECATION WARNING:\ntrack.docs: Positional argument is deprected, please use keyword argument 'image_path:' instead\neg: track.docs(image_path: #{positional_image_path_which_is_deprecated.inspect})\n"
```

Comment:
s/deprected/deprecated/

This is an excellent message. I really appreciate that it both tells you what's wrong and how to fix it.